Common/GPU/Vulkan/VulkanRenderManager.cpp

```diff
@@ -115,15 +115,17 @@ bool VKRGraphicsPipeline::Create(VulkanContext *vulkan, VkRenderPass compatibleR
 	pipe.basePipelineIndex = 0;
 	pipe.subpass = 0;
 
+	INFO_LOG(G3D, "Creating pipeline... rpType: %08x sampleBits: %d (%s)", (u32)rpType, (u32)sampleCount, tag_.c_str());
```

Comment:
This seems like it's going to be pretty noisy again.

I prefer to keep logging showing info and etc. but I don't want to get buried in 10 pages of unimportant needless logging every time I run any game.

-[Unknown]

Reply:
Oops, I'll get rid of it. Was investigating some weirdness on PowerVR.